momentumopt/colcon.pkg

```diff
@@ -0,0 +1,6 @@
+{
+    "dependencies": ["solver", "solver_lqr", "pybind11", "yaml_utils", "Eigen3"],
+    "build-dependencies": ["mpi_cmake_modules"],
+    "run-dependencies": [],
+    "test-dependencies": ["googletest-distribution"]
+}
```

Comment:
is it necessary? I checked on my mac and colcon can build the dependency tree without this. Is it different on Ubuntu?
If I run ```colcon info --packages-select momentumopt``` I get
```
  type: cmake
  name: momentumopt
  dependencies:
    build: Eigen3 GTest mpi_cmake_modules pybind11 solver solver_lqr yaml_utils
    run: Eigen3 GTest mpi_cmake_modules pybind11 solver solver_lqr yaml_utils
```